tcms/testplans/static/testplans/js/get.js

```diff
@@ -344,13 +345,13 @@ function getTestCaseExpandArea(row, testCase, permissions) {
         testCase.id,
         'TestCase.comments',
         'TestCase.remove_comment',
-        !isTestCaseConfirmed(testCase.case_status_id) && permissions['perm-delete-comment'],
+        !isTestCaseConfirmed(testCase.case_status_id, TestCaseStatusConfirmedPK) && permissions['perm-delete-comment'],
```

Comment:
if you are passing the confirmed status PK as parameter everywhere then you don't need the isTestCaseConfirmed function. Can be replaced with a simple `===` comparison.

OTOH you can make the PK a global variable like in your initial implementation and minimize the changes across the board. And IMO that is acceptable because there are other global variables already.